src/main/java/com/google/idea/perf/methodtracer/MethodTracerCommand.kt

```diff
@@ -51,9 +51,15 @@ sealed class TraceTarget {
     object PsiFinders: TraceTarget()
 
     /** Trace a specific method. */
-    data class Method(val className: String, val methodName: String?): TraceTarget()
+    data class Method(
+        val className: String,
+        val methodName: String?,
+        val parameterIndexes: List<ParameterIndex>?
+    ): TraceTarget()
 }
 
+data class ParameterIndex(val index: Int)
```

Comment:
As of now, this class isn't really needed. I just thought it would prevent breaking code elsewhere if I decided to add another property to it, but I'm also fine with using `Int` directly.